jdaviz/core/validunits.py

```diff
@@ -54,7 +54,7 @@ def create_spectral_equivalencies_list(spectral_axis_unit,
 
 def create_flux_equivalencies_list(flux_unit, spectral_axis_unit):
     """Get all possible conversions for flux from current flux units."""
-    if ((flux_unit in (u.count, (u.MJy / u.sr), u.dimensionless_unscaled))
```

Comment:
@astrofrog I had made this update to remove u.MJ / u.sr, so the list of surface brightness units should populate with all available options.